docs/basics/ExampleTodoList.md

```diff
@@ -83,14 +83,9 @@ const todo = (state = {}, action) => {
 const todos = (state = [], action) => {
   switch (action.type) {
     case 'ADD_TODO':
-      return [
-        ...state,
-        todo(undefined, action)
-      ]
+      return [...state, todo(undefined, action)]
     case 'TOGGLE_TODO':
-      return state.map(t =>
-        todo(t, action)
-      )
+      return state.map(t => todo(t, action))
```

Comment:
So, these are some examples of style violations that actually serve as a documentation aid. The multiple lines here let us break things down into smaller steps, which is helpful when you're reading through documentation and want to know exactly what's going on. I'd leave this as-is.